optimumsync/icst2025
# Contact Us Page submission

Summary of Changes:
Created a ContactUs page with dynamic contact sections for various inquiries (General, Paper Submissions, Sponsorship, etc.).
Each section displays contact details (email, phone) and optional extra content like office hours or address.
Implemented a reusable ContactSection component for each inquiry type.

Reason for the Change:
To provide a structured and easy-to-navigate Contact Us page with relevant contact details for different purposes.

How to Test:
Visit the Contact Us page and check if each section (General Inquiries, Paper Submissions, etc.) displays the correct details and links (email, phone).
Test email links and ensure phone numbers are clickable on mobile.

Additional Notes:
The page is responsive and adjusts to different screen sizes.
